lwseverity/severity.go

```diff
@@ -0,0 +1,105 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// A package for Lacework severities
+package lwseverity
+
+import (
+	"sort"
+	"strings"
+)
+
+type Severity interface {
+	GetSeverity() string
+}
+
+// Take a string representation of Lacework severity and
+// return it's normalized integer and string values
+// Critical Severity      => 1, "Critical"
+// High Severity          => 2, "High"
+// Medium Severity        => 3, "Medium"
+// Low Severity           => 4, "Low"
+// Informational Severity => 5, "Info"
+// Unknown Severity       => 0, "Unknown"
+func SeverityToProperTypes(severity string) (int, string) {
```

Comment:
Now that we have this abstracted is `Normalize` a better/simpler name for this...

Thoughts @afiune

Reply:
I went ahead with this.

Reply:
Great stuff!